river/cluster/dbstream.py

```diff
@@ -138,36 +138,38 @@ def __init__(
         minimum_weight: float = 1.0,
     ):
         super().__init__()
-        self.time_stamp = 0
+        self._time_stamp = 0
 
-        self.clustering_threshold = clustering_threshold
-        self.fading_factor = fading_factor
-        self.cleanup_interval = cleanup_interval
-        self.minimum_weight = minimum_weight
-        self.intersection_factor = intersection_factor
+        self._clustering_threshold = clustering_threshold
+        self._fading_factor = fading_factor
+        self._cleanup_interval = cleanup_interval
+        self._minimum_weight = minimum_weight
+        self._intersection_factor = intersection_factor
 
-        self.n_clusters = 0
-        self.clusters: typing.Dict[int, "DBSTREAMMicroCluster"] = {}
-        self.centers: typing.Dict = {}
-        self.micro_clusters: typing.Dict[int, "DBSTREAMMicroCluster"] = {}
+        self._n_clusters: int = 0
+        self._clusters: typing.Dict[int, "DBSTREAMMicroCluster"] = {}
+        self._centers: typing.Dict = {}
+        self._micro_clusters: typing.Dict[int, "DBSTREAMMicroCluster"] = {}
 
-        self.s: typing.Dict[int, typing.Dict[int, float]] = {}
-        self.s_t: typing.Dict[int, typing.Dict[int, float]] = {}
+        self._s: typing.Dict[int, typing.Dict[int, float]] = {}
```

Comment:
I don't think you need to make all these attributes "private". If they're not documented, people won't use them. Also, if we had to do this for every model, it would make the doc quite ugly. Can you revert this change for all the properties that don't require reclustering?